person.rb

```diff
@@ -1,5 +1,11 @@
-# class representing a person
+require_relative 'nameable'
+require_relative 'decorator'
+require_relative 'capitalize_decorator'
+require_relative 'trimmer_decorator'
+
 class Person
```

Comment:
[OPTIONAL]
Hi, your code works well, but I have an optional suggestion. You have used both inheritance and inclusion in your Person class, which could cause some issues. You could use only inheritance, and make Person a subclass of Nameable. This would make your code more consistent and avoid conflicts. To do this, change the first line of person.rb to `class Person < Nameable`. You can find more information about classes and modules here  . This is an optional change, but I think it would improve your code. Let me know if you have any questions or feedback. 😊